src/internal/LosantDevice.cpp

```diff
@@ -1,17 +1,22 @@
+#include <MQTT.h>
 #include "LosantDevice.h"
 
+MQTTClient mqttClient(MQTT_MAX_PACKET_SIZE);
```

Comment:
by doing this here, two instances of LosantDevice would share the same underlying client right? doesn't this need to be in the class?